ts/src/main/scala/ch/epfl/ts/brokers/ExampleBroker.scala

```diff
@@ -77,7 +110,12 @@ class ExampleBroker extends Component with ActorLogging {
       log.debug("Broker: received order")
       val replyTo = sender
       val uid = o.chargedTraderId()
-      val placementCost = o.costValue()
+      val placementCost = o match {
+        case mbid: MarketBidOrder => o.volume * tradingPrices(o.whatC, o.withC)._2 // we buy at ask price
```

Comment:
This case should happen only if the trader place an order before receiving any information. So it means place an order based on nothing (no indicators) and the order should also have been placed before the Broker receives any quote from the Market. So maybe we have to think how to start our Components to ensure that the market have a quote before a Trader can start trading to oust these early-no-information-based trader.